app/controllers/hosts_controller.rb

```diff
@@ -33,7 +33,7 @@ class HostsController < ApplicationController
   before_filter :validate_power_action, :only => :update_multiple_power_state
   before_filter :validate_multiple_puppet_proxy, :only => :update_multiple_puppet_proxy
   before_filter :validate_multiple_puppet_ca_proxy, :only => :update_multiple_puppet_ca_proxy
-  helper :hosts, :reports, :interfaces
+  helper :hosts, :hosts_and_hostgroups, :reports, :interfaces, :puppet_related
```

Comment:
If the order is important here what will happen with the other pagelets added later on? do they have to be added after the puppet one or just always keep them after hosts_and_hostgroups?

The reason for my question is that later on you use `super` to make sure `define_host_pagelets` isn't overridden but added to whatever was already loaded but if more pagelets are added the `super` can go to either puppet or  hosts_and_hostgroups. I'm assuming the order doesn't really matter as long as all of them are loaded but it is a little confusing to debug when super can lead to multiple paths.